src/ui/TrackViewer.tsx

```diff
@@ -291,7 +291,11 @@ export class TrackViewer extends Object2D {
         })
 
         let defaultTrackHeight = trackClasses.trackObjectClass.getDefaultHeightPx != null ? trackClasses.trackObjectClass.getDefaultHeightPx(model) : 100;
-        let expandable = trackClasses.trackObjectClass.getExpandable != null ? trackClasses.trackObjectClass.getExpandable(model) : true;
+
+        let expandable =
+            (trackClasses.trackObjectClass.getExpandable != null ? trackClasses.trackObjectClass.getExpandable(model) : true)
```

Comment:
@emro I know George uses "!="; but is there a reason why you are using it too rather than "!==". Using "!=" gets tricky like 
```
1 !== '1'
true

1 !== 1
false
```

Reply:
I spoke with Emma. This code is code she copied from George.